frontend/src/components/Charts/nivoScheme.ts

```diff
@@ -0,0 +1,29 @@
+import { Theme } from "@nivo/core"
+
+export const nivoScheme: Theme = {
+  axis: {
+    ticks: {
+      text: {
+        fill: "rgb(255, 255, 255)"
```

Comment:
Would be great if these variables can be dynamically set to match the colors defined in the MUI theme in `App.js` . It is provided as context to all children. https://github.com/Reckless-Satoshi/robosats/blob/8eda1dd20fec576726cb1578e263bfb43472be48/frontend/src/components/App.js#L26-L42
This would solve the conflict we have now with the light/dark themes as well as make it future-proof for any possible theme change.

Take a look at MUI Theming and the use of the hook `useTheme()` https://mui.com/material-ui/customization/theming/